### ISSUE TITLE
NameError: name 'selective_scan_cuda_core' is not defined. Did you mean: 'selective_scan_flop_jit'?

### ISSUE THREAD

**Issue by wsasdsda** [MzeroMiko/VMamba#147]
Hi, thans for your contribution!
I get "NameError: name 'selective_scan_cuda_core' is not defined. Did you mean: 'selective_scan_flop_jit'? "
How to fix it?

**Comment by cyc1111111111** [comment]
I have the same problem, and I don't think this can be solved with comments,

> 嗨，谢谢你的贡献！我收到“NameError：未定义名称'selective_scan_cuda_core'。你的意思是：“selective_scan_flop_jit”吗？“ 怎么解决？

**Comment by cyc1111111111** [comment]
> 嗨，谢谢你的贡献！我收到“NameError：未定义名称'selective_scan_cuda_core'。你的意思是：“selective_scan_flop_jit”吗？“ 怎么解决？

I am also having the same problem, do I know how to solve it, I am running under Windows environment

**Comment by MzeroMiko** [comment]
have you installed the package in `kernels/selective_scan`?

**Comment by wsasdsda** [comment]
> have you installed the package in `kernels/selective_scan`?

Yes, I execute "cd kernels/selective_scan && pip install." , and it has been successfully installed.

**Comment by dragonBrother1** [comment]
请问这一问题当前解决了吗？

**Comment by dragonBrother1** [comment]
I want to know how to fix it
